tests/test_recipes/test_core_recipe.py

```diff
@@ -0,0 +1,141 @@
+"""
+
+@kylel
+
+"""
+
+import os
+import unittest
+
+from papermage.recipes import CoreRecipe
+from papermage.types import Entity, Document, Image
+from tests.test_recipes.core_recipe_fixtures import (
+    BASE64_PAGE_IMAGE,
+    FIRST_3_BLOCKS_JSON,
+    FIRST_5_ROWS_JSON,
+    FIRST_10_TOKENS_JSON,
+    FIRST_10_VILA_JSONS,
+    FIRST_1000_SYMBOLS,
+    PAGE_JSON,
+    # SEGMENT_OF_WORD_JSONS,
```

Comment:
I left the word field tests commented out in case we want to add word tokenization back into the core recipe later.